core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -35,7 +35,9 @@
             <p-button class="p-button-sm" label="Save" />
         </ng-container>
     </dot-edit-ema-toolbar>
-    <div class="iframe-wrapper">
+    <div
+        class="iframe-wrapper"
+        [ngStyle]="{ 'grid-column': es.haveEnterpriseLicense ? '' : '1/-1' }">
```

Comment:
What's the idea here?

Reply:
Is for dont show the palette and expand the editor when the user dont have license
But yep, we need to discuss about it, bcs i remember the user dont have access to entire edit-ema if dont have license, so that validation on palette is unnecessary

Reply:
This should be responsive by default. If the palette is not there, the iframe should just grow.